eslint-config-base/index.js

```diff
@@ -102,6 +102,14 @@ module.exports = {
       message: "`with` is disallowed in strict mode because it makes code impossible to predict and optimize.",
     }],
 
+    // Leaving out the await can hide the fact that you're not catching thrown errors:
+    // https://twitter.com/_jayphelps/status/1324565522755788803
+    // Additionally, it will prevent the function from being part of the stack trace
+    // if not catching them:
+    // https://eslint.org/docs/rules/no-return-await
+    // This does not outweigh the minor performance benefit, in most cases.
+    "no-return-await": ["off"],
```

Comment:
That's an interesting thread, but it seems to talk about `return await x()` inside of a `try ... catch`, which `no-return-await` doesn't prevent [as per the doc](https://eslint.org/docs/rules/no-return-await).

It would be good to keep the error if `return await` is used in general and find one that throws when **not** `return await`ing inside a `try ... catch`. But while turning off this rule disables the former, which I think is a good rule, it doesn't prevent the latter, which would be desirable.

Am I missing something?

Reply:
Oh ha, good point, I'd missed that in the docs. It doesn't look like it actually errors when you do a `return` _without_ `await` in a `try ... catch` block though, in which case I think I'd still be leaning to disabling this rule - if I'm being conditioned to leave out the `await` on `return`s, then I'd expect it to be likely that I'd forget that this is an exception. (And in general, I'd prefer the consistency.)

But I would certainly be fine with keeping the rule given that it does allow it in a `try ... catch` block. What do you think?